nameless_api.conf

```diff
@@ -2,9 +2,10 @@
 server {
   # Listen for incoming connections on port 80 (HTTP traffic)
   listen 80;
-  # Set the server name for this block to "nameless.api"
-  server_name nameless.api;
-  # Define a new location block for the root path ("/")
+  # Set the server name for this block to relativepath
+  server_name relativepath;
+
+  # Define a location block for the static page app at the root path ("/")
   location / {
     # Forward incoming requests to a server running on localhost:3000
     proxy_pass http://localhost:3000;
```

Comment:
So you're serving the static content from node. What if you wanted to server the static content from nginx?

Reply:
@stephenlauck  
This is the config for the api app. I have a separate relativepath.conf for the static site, not sure if thats the protocol. Which is preferred, one .conf for both locations or separate .conf files for each location? I was thinking the latter.

Reply:
There are two places you can serve static content: 1. from your upstream node app 2. from nginx directly. If you were to serve from nginx directly, you can put your static content into `/var/www/relativepath` and then point the location to that path. This means that nginx isn't even going upstream for the static but is serving it directly. Let me know if this makes sense.

Reply:
I remember serving the viking app at /var/www/viking with nginx, that makes sense. 

The reverse proxy and upstream etc are confusing. I'll search for a visual of these so I can better wrap my head around it.